pkg/gui/branches_panel.go

```diff
@@ -62,20 +62,28 @@ func (gui *Gui) handleNewBranch(g *gocui.Gui, v *gocui.View) error {
 }
 
 func (gui *Gui) handleDeleteBranch(g *gocui.Gui, v *gocui.View) error {
+	gui.deleteBranch(g, v, false)
```

Comment:
You need to say `return gui...` here so that you're returning an error object as the function signature asserts (same with handleForceDeleteBranch)

Reply:
D'oh, that makes sense. My bad.